jolpica/formula_one/data_correction.py

```diff
@@ -35,3 +40,22 @@ def run_data_correction():
         entry.is_eligible_for_points = False
         updated.append(entry)
     SessionEntry.objects.bulk_update(updated, fields=["position", "is_eligible_for_points"])
+
+
+def session_correction():
+    corrections = {
+        SessionType.RACE: "16:00:00",
+        SessionType.SPRINT_RACE: "14:00:00",
+        SessionType.QUALIFYING_ONE: "18:00:00",
+        SessionType.QUALIFYING_TWO: "18:00:00",
+        SessionType.QUALIFYING_THREE: "18:00:00",
+    }
+
+    qatar_2024 = Session.objects.filter(round__season__year=2024, round__number=23, type__in=corrections.keys())
+
+    updates = []
+    for entry in qatar_2024:
```

Comment:
I believe this should be called a session not an entry.

Reply:
Updated and rebased since it was a minor change